x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportPutDataFrameAnalyticsAction.java

```diff
@@ -179,7 +182,14 @@ private void updateDocMappingAndPutConfig(DataFrameAnalyticsConfig config,
             client,
             clusterState,
             ActionListener.wrap(
-                unused -> configProvider.put(config, headers, listener),
+                unused -> configProvider.put(config, headers, ActionListener.wrap(
+                    indexResponse -> {
+                        auditor.info(
+                            config.getId(),
+                            Messages.getMessage(Messages.DATA_FRAME_ANALYTICS_AUDIT_CREATED, config.getAnalysis().getUserFriendlyName()));
```

Comment:
I'm not sure we need to make the effort to convert to friendly names here. Users will be specifying those in the config with the writeable name so why not use that in the audit message too? That way we don't need `getUserFriendlyName()` at all.